src/style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars > * {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 > .stars__star--1 {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 > .stars__star--1,
+.stars--2 > .stars__star--2 {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 > .stars__star--1,
+.stars--3 > .stars__star--2,
+.stars--3 > .stars__star--3 {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 > .stars__star--1,
+.stars--4 > .stars__star--2,
+.stars--4 > .stars__star--3,
+.stars--4 > .stars__star--4 {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 > * {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
All of this can be implemented in 7 rows😉 I will give you a hint:
`.stars--1 .stars__star:nth-child(-n+1)`

Reply:
Have done it in 7 rows and it looks fantastic :) Thanks! I had forgotten about this trick, although it was there in videos.